dev-util/bitcoin-tx/files/0.21.0-compat-libsecp256k1-0.1_pre20210628.patch

```diff
@@ -0,0 +1 @@
+../../../net-p2p/bitcoind/files/0.21.0-compat-libsecp256k1-0.1_pre20210628.patch
```

Comment:
This isn't portable behaviour and nothing in PMS says it must work. You have to duplicate the patch or, alternatively, put it in a tarball you re-use (or a distfile you download with the same name).

Reply:
Does Gentoo even run on any platforms that don't support symbolic links? 🙄 At some point, trying to cater to completely broken environments means you're chopping off your own feet.

But okay, regardless, I don't have any way of getting a tarball into Gentoo's distribution mirrors, so I guess I'm stuck wasting space on every Gentoo user's file system by duplicating these patches. Will do. 😒

Reply:
It's more that the layout isn't guaranteed:
- https://blogs.gentoo.org/mgorny/2017/03/17/why-you-cant-rely-on-repository-format-pms/
- https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#portdir-and-eclassdir-are-removed

IIRC rsync doesn't handle it well either for repository syncs.

Reply:
But all that said, I think I'd rather we just stable the new Bitcoins and cleanup, it's way less hassle. What do you think?

Reply:
> * https://blogs.gentoo.org/mgorny/2017/03/17/why-you-cant-rely-on-repository-format-pms/

Makes a good case, though I do believe that PMS requires a package manager to make all the files in the `files` repository subdirectory available in `${FILESDIR}` during execution of `src_*` phase functions. Maybe the argument is that the main Gentoo repo, despite being authored as a file-system tree, might not be delivered onto a user's system in that same format? I guess that's fair enough.